maintainers/scripts/haskell/upload-nixos-package-list-to-hackage.sh

```diff
@@ -19,3 +19,4 @@ package_list="$(nix-build -A haskell.package-list)/nixos-hackage-packages.csv"
 username=$(grep "^username:" ~/.cabal/config | sed "s/^username: //")
 password_command=$(grep "^password-command:" ~/.cabal/config | sed "s/^password-command: //")
 curl -u "$username:$($password_command | head -n1)" --digest -H "Content-type: text/csv" -T "$package_list" http://hackage.haskell.org/distro/NixOS/packages.csv
+echo
```

Comment:
If this `curl` command succeeds, it returns `OK!` without an ending newline.  Adding an `echo` here makes sure an ending newline is added.

Without adding this `echo`, the output from `merge-and-open-pr.sh` gets slightly messed up since there is no newline.